core/src/main/java/org/apache/calcite/sql/dialect/SparkSqlDialect.java

```diff
@@ -70,6 +70,10 @@ public SparkSqlDialect(SqlDialect.Context context) {
     return true;
   }
 
+  @Override public boolean supportsNestedAggregations() {
+    return false;
+  }
+
```

Comment:
IMO, the title of this issue is a little confusing. What I get from the title is BigQuery/Hive/Spark should support nested aggregate, but we don't. However, the fact is BigQuery/Hive/Spark support nested aggregate wrongly. So I suggest changing it. What about `BigQuery, Hive, Spark SQL dialects support nested aggregates wrongly`?

Reply:
+1

Or `BigQuery, Hive, Spark SQL dialects claim they support nested aggregates wrongly`

Reply:
@chunweilei and @amaliujia  point duly noted. Would it be right to say that ### BigQuery, Hive, Spark SQL dialects do not explicitly support nested aggregates?

Reply:
+1 for the new title.